.lintstagedrc

```diff
@@ -1,3 +0,0 @@
-{
```

Comment:
we should keep this. It's how we defined what files need to be formatted per commit